netlify/functions/favorites.ts

```
@@ -0,0 +1,32 @@
+      if (!userId) return { statusCode: 400, body: 'Missing userId' };
+      const favorites = await storage.getUserFavorites(parseInt(userId), type);
+      return { statusCode: 200, body: JSON.stringify(favorites) };
+    }
+    if (event.httpMethod === 'POST') {
+      const body = event.body ? JSON.parse(event.body) : {};
+      const favoriteData = insertFavoriteSchema.parse(body);
+      const favorite = await storage.addFavorite(favoriteData);
+      return { statusCode: 200, body: JSON.stringify(favorite) };
+    }
+    if (event.httpMethod === 'DELETE') {
+      const userId = event.queryStringParameters && event.queryStringParameters.userId;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)